Colours_And_Fonts_style_v1.css

```diff
@@ -22,23 +24,86 @@ h2 {
 }
 
 h3 {
-  color: #000000;
-  font-size: 20px;
+  color: #FFF;
+  font-size: 45px;
   text-align: center;
+  margin-top: 30px;
+  margin-left: 150px;
 }
 
 h4 {
+  color: #FFF;
+  font-size: 45px;
+  text-align: center;
+  margin-top: 30px;
+  margin-left: 1100px;
+}
+
+h5 {
   color: #000000;
   font-size: 30px;
   text-align: center;
+  margin-left: 500px;
 }
-	
-p {
-  font-size: 20px;
-  font: split;
+
+/* Paragraph Styles */
+.para1, .para2 {

```

Comment:
I made two sepreate paragraph classes as evrey page is different for their parargraphs so I added different paragraphs in my css.